languageserver/go.mod

```diff
@@ -11,3 +11,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	google.golang.org/grpc v1.32.0
 )
+
+replace (
+	github.com/onflow/cadence => ../
+)
```

Comment:
Once the separate PR for the parser changes has been merged, we can update this PR and replace these lines with a version bump to the new Cadence version that includes the parser changes.